app/Http/Controllers/Admin/PermissionGroupController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Admin\PermissionGroupRequests;
+use App\Repositories\Admin\PermissionGroup\PermissionGroupRepositoryInterface as PermissionGroupRepository;
+
+class PermissionGroupController extends Controller
+{
+    protected $permissionGroupRepository;
+
+    public function __construct(PermissionGroupRepository $permissionGroupRepository)
+    {
+        $this->permissionGroupRepository = $permissionGroupRepository;
+    }
+
+    public function index()
+    {
+        return view('admin.permission-group.index', [
+            'permissionGroups' => $this->permissionGroupRepository->paginate(),
+        ]);
+    }
+
+    public function show($id)
+    {
+        $permissionGroup = $this->permissionGroupRepository->findById($id);
```

Comment:
if (!$permissionGroup = $this->permissionGroupRepository->save($request->validated())) {
abort(404);
}